### PR TITLE
feat(apple): Update installation with SPM

### PR DESCRIPTION
**Pull request by brustolin**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes
Added extra information on how to install Sentry SDK with SPM.

### CODE REVIEW

File: src/platforms/apple/common/install/swift-package-manager.mdx [hunk 1]
```diff
@@ -10,6 +10,16 @@ https://github.com/getsentry/sentry-cocoa.git
 ```
 
 You can define your dependency rule by selecting the SDK version (or branch), and then click the "Add Package" button.
+You will then be prompt to choose between 3 options `Sentry`, `Sentry-Dynamic` and `SentrySwiftUI`.
+
+- `Sentry` is the static framework, which is the recommended option if you prefer a fast app start time.
+- `Sentry-Dynamic` is the dynamic framework, which is the recommended options if you more control.
```
Comment (kahest) [inline src/platforms/apple/common/install/swift-package-manager.mdx:16]:
```suggestion
- `Sentry-Dynamic` is the dynamic framework, which is the recommended option if you require more control.
```

Comment (kahest) [reply src/platforms/apple/common/install/swift-package-manager.mdx:16]: what kind of control?

File: src/platforms/apple/common/install/swift-package-manager.mdx [hunk 2]
```diff
@@ -10,6 +10,16 @@ https://github.com/getsentry/sentry-cocoa.git
...
+- `SentrySwiftUI` is an addicional library if you're using SwiftUI in your project and wants to track performance of your views.
```
Comment (kahest) [inline src/platforms/apple/common/install/swift-package-manager.mdx:17]:
```suggestion
- `SentrySwiftUI` is used to track performance of SwiftUI views, see more information in the [docs](https://docs.sentry.io/platforms/apple/guides/ios/performance/instrumentation/swiftui-instrumentation/).
```

**Review by kahest — CHANGES_REQUESTED** [review]
left some suggestions

File: src/platforms/apple/common/install/swift-package-manager.mdx (hunk 1 above)
Comment (brustolin) [reply src/platforms/apple/common/install/swift-package-manager.mdx:16]: Good question. It did more sense in my mind.

File: src/platforms/apple/common/install/swift-package-manager.mdx [hunk 3]
```diff
@@ -10,6 +10,16 @@ https://github.com/getsentry/sentry-cocoa.git
 ```
 
 You can define your dependency rule by selecting the SDK version (or branch), and then click the "Add Package" button.
+You will then be prompted to choose one of the options `Sentry`, `Sentry-Dynamic` or `SentrySwiftUI`.
+
+- `Sentry` is the static framework, which is the recommended option if you prefer a fast app start time.
+- `Sentry-Dynamic` is the dynamic framework, which is the recommended option if you require more control.
```
Comment (brustolin) [inline src/platforms/apple/common/install/swift-package-manager.mdx:16]:
```suggestion
- `Sentry-Dynamic` is the dynamic framework.
```

**Review by kahest — APPROVED** [review]

**Pull request merged**